src/nplinker/genomics/antismash/antismash_downloader.py

```diff
@@ -322,7 +323,87 @@ def _extract_antismash_zip(antismash_obj, project_file_cache):
 
     return True
 
-def download_antismash_data(genome_records, project_download_cache, project_file_cache):
+def download_and_extract_antismash_data(item_id, download_root, extract_path): 
```

Comment:
The function is still coupled with PODP pipeline (the `genome_status.txt`), let's have a chat.